src/clients/ldapkrb.rb

```diff
@@ -24,4 +24,4 @@
 require 'authui/main_dialog'
 
 Auth::AuthConfInst.read_all
-Auth::MainDialog.new(:ldapkrb).run
+Auth::MainDialog.new(:ldapkrb, 'LDAP and Kerberos Client').run
```

Comment:
the constructor accepts untranslated string, and then translate the string before use. I'm just trying to avoid including i18n here.